pedpy/methods/method_utils.py

```diff
@@ -872,6 +880,76 @@ def _compute_movememnt_adaptive_border(
     return result[result.window_size > 0]
 
 
+def _compute_individual_movement_acceleration(
+    *,
+    traj_data: TrajectoryData,
+    frame_step: int,
+    acceleration_border_method: AccelerationCalculation = AccelerationCalculation.BORDER_EXCLUDE,
+) -> pandas.DataFrame:
+    if acceleration_border_method == AccelerationCalculation.BORDER_EXCLUDE:
+        return _compute_movement_acceleration_exclude_border(
+            traj_data, frame_step
+        )
+
+    raise ValueError("speed border method not accepted")
```

Comment:
Copy Paste Error

Reply:
I changed the ValueError Text. Is it ok now?

Reply:
Yes, this is what I meant